dockerfiles/tasks.py

```diff
@@ -157,3 +160,33 @@ def buildassets(c):
     """Build all assets for the application and push them to backend storage"""
     c.run(f'docker-compose -f {DOCKER_COMPOSE_ASSETS} run --rm assets bash -c "npm ci && node_modules/bower/bin/bower --allow-root update && npm run build"', pty=True)
     c.run(f'{DOCKER_COMPOSE_COMMAND} run --rm web python3 manage.py collectstatic --noinput', pty=True)
+
+
+@task(help={
+    'action': 'Action to realize on Transifex ("pull" or "push")',
+})
+def translations(c, action):
+
+    if action not in ('pull', 'push'):
+        print(f'Action passed ("{action}") not supported. Use "pull" or "push".')
+        sys.exit(1)
+
+    transifex_token = os.environ.get('TRANSIFEX_TOKEN', None)
+    if not transifex_token:
+        print('You need to export TRANSIFEX_TOKEN environment variable.')
+        sys.exit(1)
+
+
+    # Install Transifex Client in Docker container
+    download_file = 'https://github.com/transifex/cli/releases/download/v1.1.0/tx-linux-amd64.tar.gz'
+    c.run(f'{DOCKER_COMPOSE_COMMAND} run --rm web /bin/bash -c "curl --silent --location {download_file} | tar --extract -z --file=- tx"', pty=True)
```

Comment:
Not sure what happened, but this command froze and killed the task when I tried first. With `--silent`, I didn't know what was happening. Is there are reason we want `--silent` here? If curl output doesn't play well with pty output, at least some note that a download was started would be helpful.